lms/templates/navigation.html

```diff
@@ -124,15 +124,21 @@ <h2 class="course-header"><span class="provider">${course.display_org_with_defau
         % endif
       </%block>
       % if not settings.FEATURES['DISABLE_LOGIN_BUTTON']:
+          % if settings.FEATURES.get('ENABLE_COURSE_DISCOVERY'):
```

Comment:
Rather than checking settings directly here, it would be better to create a context parameter such as 'course_discovery_enabled' and then determine the status of this value in the view function